### DISCUSSION TITLE
Stucked on step 4 (not updated to step 5)

### DISCUSSION THREAD

**Discussion by davidcorcuera** [skills/.github#466] [Code with Codespaces] [answered]
Hello, I've arrived to and complete step 4 and the train appears when running /usr/games/sl. Then I refresh the isnstructions page (as requested after 20 seconds) to continue to step 5 but nothing updates. Any idea, please?
Thanks

**Comment by davidcorcuera** [comment, marked as answer]
I answer myself. It seems step 4 is last step and after some minutes it updated to Finish step which is a recap.

**Reply by anieshak** [reply]
I am getting error - sl: command not found while trying to execute step 4 of the Activity - 
**Code with GitHub Codespaces and Visual Studio Code**

![sl command not found](https://github.com/skills/.github/assets/20052332/ae95647b-681a-46b2-a57b-9d47122fcde1)
 

**Verify your codespace is running, as you did previously.** - I did

**Verify the setup.sh file is present in your VS Code editor.**  - I did

**< I was able to verify that the above files were present yet the command (sl) doesn't seem to function.>**

From the VS Code terminal, type or paste:

sl
Enjoy the show!

**I am unable to realize what this command is supposed to do? Can someone please help as to what the output should be. I wish Github pasted screenshots of what the expected output was supposed to look like.**

**Reply by tomfp** [reply]
It does not work for me either. 
I think enabling the dotfiles step is supposed to run setup.sh when your codespace starts up
But it doesn't do that.  
You can run setup.sh from the terminal in the root directory 
> ./setup.sh (after the fix above)
But still the alias sl is not set up
If you type the last line at the terminal, you can set up the alias manually
To see if its set up 
> type sl
Should show: sl is aliased to `/usr/games/sl'
> sl
After all that you can see an ASCII animation

**Reply by Lua-Matlab-Python-R-J2EE** [reply]
Before running the sl command in the terminal, type bash and press Enter. Then, type the sl command.

**Reply by Besho777** [reply]
/Setup.sh

**Comment by b2kay** [comment]
I think there must be 2 steps before running sl.
1. run setup.sh on vs terminal " ./setup.sh"
2. source bashrc: "source .bashrc"

then you can run sl on vs terminal

**Comment by Besho777** [comment]
> I answer myself. It seems step 4 is last step and after some minutes it updated to Finish step which is a recap.
>
